practice-one/src/components/Account/Account.css

```diff
@@ -0,0 +1,24 @@
+.account {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 18px;
+    padding-bottom: 30px;
+}
+
+.account__avatar {
+    width: 45px;
+    height: auto;
+    border-radius: 12px;
```

Comment:
why the border-radius is 12px?

Reply:
Because when I check the design, this image has a border like that, so I gave the parent tag to wrap the image with a border like that.
![image](https://github.com/datdragon777/React-Training/assets/74806826/8611958a-fece-4ff4-b453-d94ca95e057f)